zeppelin-web/README.md

```diff
@@ -12,49 +12,49 @@ All build commands are described in [package.json](./package.json)
 
 ```sh
 # install required depepdencies and bower packages (only once)
```

Comment:
seems there are fews place still refers `bower`, could you please do a global search to find them out?